### PR TITLE
Fix miscellaneous phrasing issues and invalid HTML

### PR DESCRIPTION
**Pull request by WasabiFan**
These were some phrasing oddities that I noticed while looking over etcher.io a few days ago. There were also a fair number of HTML validation issues, so I changed those as well (that makes me think that this isn't auto-generated :laughing:). Some of the copy changes are pretty subjective, so I won't be offended if you don't want them. This PR branch can be pushed to by repo maintainers.

One of the changes was to expand the tagline to be the grammatically-correct version that now shows up in the README. However, I kept the ampersands as they were on this site before, even though the other version uses the word "and." You may want to change that.

### CODE REVIEW

**Comment by lurch** [comment]
I guess the `og:description` should be updated too?
I think the website tagline ought to match https://github.com/resin-io/etcher and https://github.com/resin-io/etcher/blob/master/README.md and https://github.com/resin-io/etcher/blob/master/package.json ;-)

**Comment by WasabiFan** [comment]
Good point -- I hadn't noticed that one. Do you think that it should be normalized to ampersands or "and"s? Grammatically, I think the latter is the most widely-accepted.

**Comment by WasabiFan** [comment]
Bump -- mentioning @craig-mulligan

**Comment by craigmulligan** [comment]
Hey @WasabiFan thanks for the help :)! Unfortunately `index.html` is generated from `template.html` so we'll need to make the changes there, then run `gulp` to rebuild `index.html`. Mind updating the readme to point people to `template.html` instead?

**Comment by WasabiFan** [comment]
Haha, I figured that was going to be the case :laughing: I'll try to figure out where these changes should be and amend this PR.

**Comment by lurch** [comment]
> Do you think that it should be normalized to ampersands or "and"s?

I think it should be normalized to match the other files I pointed to :-)

**Comment by WasabiFan** [comment]
I updated the PR so that the changes are in the proper files. Make sure that you squash if/when merging.

File: template.html [hunk 1]
```diff
@@ -99,7 +99,7 @@
               href="https://twitter.com/share"
               data-size="large"
               data-url="http://www.etcher.io"
-              data-text="Meet Etcher by @resin_io an awesome new way to write SD cards. Give it a shot and you'll never go back!">
+              data-text="Meet Etcher by @resin_io, an awesome new way to write SD cards. Give it a shot and you'll never go back!">
```
Comment (lurch) [inline template.html:102]: Could placing the comma here accidentally confuse some twitter clients into thinking it's part of the username? \*shrug\*
(I'm a non-twitter-user myself)

File: template.html [hunk 2]
```diff
@@ -197,7 +197,7 @@
           <div class="col-md-6 promo">
             <img class="logo" src="static/images/<= logo.nav =>.png" srcset="static/images/<= logo.nav =>@2x.png 2x"/>
             <p>
-              Etcher is an open source project by <a class="resin-link" href="https://resin.io">resin.io</a> - Modern DevOps for the Industrial Internet of Things
+              Etcher is an open-source project by <a class="resin-link" href="https://resin.io">resin.io</a> - Modern DevOps for the Industrial Internet of Things
```
Comment (lurch) [inline template.html:200]: This hyphen should probably be left out, to match https://github.com/resin-io/etcher/blob/master/lib/gui/index.html#L44 ?

File: template.html [hunk 3]
```diff
@@ -7,7 +7,7 @@
     <meta property="og:url" content="http://www.etcher.io" />
     <meta property="og:type" content="website" />
     <meta property="og:title" content="Etcher by Resin.io" />
-    <meta property="og:description" content="Burn images to SD cards and USB drives, safe and easy." />
+    <meta property="og:description" content="Burn images to SD cards & USB drives, safely and easily." />
```
Comment (lurch) [inline template.html:10]: In the spirit of DRY, could this also come from `data.json`? Or doesn't that make sense in this case?

**Comment by craigmulligan** [comment]
We have completely rewritten the site, so closing this. Thanks for the effort though 👍

**Pull request closed without merging**